### ISSUE TITLE
HealthDeathWarning; add wiki entry?

### ISSUE THREAD

**Issue by czarchasm00** [Hypnootize/m0rehud#37]
HealthDeathWarning is an attribute (?) inside of HudPlayerHealth_Base.res. This is a changeable percentage value. Whenever your health hits this value, your health will turn to the color stated in HealthDeathWarningColor. This acts as a warning system so that if your health gets below a certain percent (by default 49% of your class's max health), it will change the color of your health to better alert you of being in a dangerous situation.

However, this 49% default value does not fit all scenarios. As a medic player, this 49% value will have the HealthDeathWarning color activate at 74HP. This is far too late, as at 75-100ishHP I can be oneshot by Scout, Soldier, Demoman, and Engineer (and obv Sniper and Spy). This can result in me not using my uber to save myself because I wasn't aware of how low my HP was.

This is hugely important for competitive play especially for medic players. In 6s, the highest damage that can be done to me in a single shot is from Soldier's Rocket Launcher. It deals a maximum of 112 damage. By putting my HealthDeathWarning at 76%, now the color will change at 114; two health points above Soldier's maximum damage. As someone who played two seasons of competitive Medic, [modifying this value](https://github.com/czarchasm00/m0rehud-cz/commit/ca4990f8445d162f27d2c02d36c3b5f26e2d163a) changed the outcome of games.

Even for casual players, if they happen to main Medic or another class, this could be an option they'd prefer to know about and be able to do.

I highly recommend this be added to the wiki.

**Comment by Hypnootize** [comment]
This is actually something I meant to add as a customization in the beginning as I also believe it to be fairly important. 
I ended up dropping some of these extra customizations as I first wanted to thoroughly test the customizations panel to make sure everything worked and was in order, being the first time I have worked with this method. 
Once I'm confortable with it I'll start adding new content, so I might very well add this soon, but in the mean time I'll happily add this to the health wiki page when I can, thanks for the suggestion!
